__tests__/index.ts

```diff
@@ -277,7 +277,17 @@ describe('copy', () => {
     expect(result.array[0]).toBe(cloneReusedObject);
     expect(result.array[1]).not.toBe(reusedObject);
     expect(result.array[1]).toBe(cloneReusedObject);
-  })
+  });
+  
+  it('will copy an object with a constructor property', () => {
```

Comment:
In the vein of my other comment, I would recommend checking that the prototypes of the original and copy are the same. I would also create a similar test, but with a custom object instead of a standard POJO.

Reply:
Added a test case with a new class `Bar` that holds a string at `constructor`. I started by writing the second test to see that my current approach fails and then fixed it with your suggestion.